Automattic/jetpack
# Charts: Fixing package TS build

Fixes n/a

## Proposed changes:

* add qa build step generating TS files
 
v0.1.0 of the package had broken TS support, making it harder to use in other projects. Adding a build step fixes this issue

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* checkout the branch and navigate to the directory: `cd projects/js-packages/charts`
* run `pnpm run build:prod`
* verify that there are no errors and files are created in the dist folder
* verify that the existing components are not broken by running storybook:`pnpm run storybook`
